airflow/providers/openlineage/provider.yaml

```
@@ -48,6 +48,7 @@ versions:
 dependencies:
   - apache-airflow>=2.8.0
   - apache-airflow-providers-common-sql>=1.6.0
+  - apache-airflow-providers-common-compat>=1.1.0
```

```suggestion
  - apache-airflow-providers-common-compat>=1.2.0
```

Doesn't change much but just to have less versions to look on